marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-view.tsx

```diff
@@ -139,6 +149,22 @@ const GraphView: React.FC<Props> = (props) => {
     </span>
   </HCButton>;
 
+  const revertButton = <HCButton
+    className={props.canWriteEntityModel ? (!modelingOptions.isModified ? styles.disabledPointerEvents : "") : styles.disabledPointerEvents}
+    disabled={props.canWriteEntityModel ? !modelingOptions.isModified : true}
+    aria-label="revertChangesGraphView"
```

Comment:
I noticed publish aria label pattern are different. Should we use the same pattern for aria labels? I don't know if the standard would be camel case or the snake case.

Reply:
@aniamy I can see snake case in this file; I will change and keep it consistent